app/helpers/form_helper.rb

```diff
@@ -104,27 +104,31 @@ def radio_button_f(f, attr, options = {})
     end
   end
 
-  def select_f(f, attr, array, id, method, select_options = {}, html_options = {})
+  def select_include_blank(array, include_blank, html_options, select_options)
+    unless array.kind_of?(Array)
+      return [array, select_options, html_options]
+    end
```

Comment:
I'm not sure if this should/can handle "ActiveSupport::SafeBuffer" like: `<option value=\"\"></option>\n<option data-suggest_new=\"false\" value=\"1\">subnet-test (127.0.0.1/24)</option>`

Reply:
https://github.com/theforeman/foreman/blob/24634a9023e2f8ecc2d9db9e1c1490c96f8e3de7/app/helpers/hosts_nic_helper.rb#L17-L24